web/thesauruses/clojure/1/operators.json

```diff
@@ -0,0 +1,213 @@
+{
+  "meta": {
+    "language": "clojure",
+    "language_version": "1.10.3",
+    "language_name": "Clojure",
+    "structure": "operators"
+  },
+  "concepts": {
+    "addition": {
+      "code": "(+ 1 2 3)",
+      "name": "Addition operator"
+    },
+    "addition_assignment": {
+      "not-implemented": "true",
+      "name": "Addition and assignment operator"
+    },
+    "subtraction": {
+      "code": "(- 10 4)",
+      "name": "Subtraction operator"
+    },
+    "subtraction_assignment": {
+      "not-implemented": "true",
+      "name": "Subtraction and assignment operator"
+    },
+    "multiplication": {
+      "code": "(* 12 12)",
+      "name": "Multiplication operator"
+    },
+    "multiplication_assignment": {
+      "not-implemented": "true",
+      "name": "Multiplication and assignment operator"
+    },
+    "division": {
+      "code": "(/ 1024 16)",
+      "name": "Division operator"
+    },
+    "division_assignment": {
+      "not-implemented": "true",
+      "name": "Division and assignment operator"
+    },
+    "integer_division": {
+      "comment": "Clojure includes a Ratio type to represent rational ratios exactly, but this can be coerced to an int",
+      "code": "(int (/ 10 3))",
+      "name": "Integer division operator"
+    },
+    "integer_division_assignment": {
+      "not-implemented": "true",
+      "name": "Integer division and assignment operator"
+    },
+    "unary_plus": {
+      "comment": "All arithmetic operators can take arbitrarily many arguments, including `+`, but this provides unclear utility",
+      "code": "(+ 5)",
+      "name": "Unary plus operator"
+    },
+    "unary_minus": {
+      "comment": "All arithmetic operators can take arbitrarily many arguments, including `-`, but this provides unclear utility",
+      "code": "(- 10)",
+      "name": "Unary minus operator"
+    },
+    "modulus": {
+      "code": "(mod 18 8)",
+      "name": "Modulus (remainder) operator"
+    },
+    "modulus_assignment": {
+      "not-implemented": "true",
+      "name": "Modulus and assignment operator"
+    },
+    "increment": {
+      "code": "(inc 1)",
+      "name": "Increment (add 1) operator"
+    },
+    "decrement": {
+      "code": "(dec 10)",
+      "name": "Decrement (subtract 1) operator"
+    },
+    "exponential": {
+      "not-implemented": "true",
+      "name": "Exponential operator"
+    },
+    "factorial": {
+      "comment": "No built in function, but easy enough to write.",
+      "code": "(defn ! [n]\n  (reduce *' (range 1 (inc n))))",
+      "name": "Factorial operator"
```

Comment:
My first thought is this is a custom-implementation and shouldn't be here. If it's not built-in, it's `not-implemented: true`. 

Second, I always hate using things like "this is easy" because you don't know the experience level of the person using the tool. It's "easy" to someone like me and you, but might not be to someone newer to Clojure or programming.

Reply:
Done.